clang/lib/Interpreter/Interpreter.cpp

```diff
@@ -521,6 +618,23 @@ llvm::Error Interpreter::CreateExecutor() {
     return llvm::make_error<llvm::StringError>("Operation failed. "
                                                "No code generator available",
                                                std::error_code());
+#ifndef _WIN32
```

Comment:
We must check the target triple, not an ifdef because we can be cross compiling.